spec/features/devise/registration_spec.rb

```diff
@@ -0,0 +1,42 @@
+feature "Registration" do
```

Comment:
Can we continue the commenting pattern as here:
  https://github.com/awong-dev/SCIInterim/blob/master/spec/features/patients/registry_spec.rb#L1

Curious to see how onerous it is.  The idea of writing these comments is it forces the spec to stay focused on particular assertion.

If they end up just repeating what we write in the scenario text too much we can drop it...but for now would like to push the trend a little while longer.

Example of such assertions being used to force focus on tests in my previous project:
  https://github.com/adobe/chromium/blob/master/base/bind_unittest.cc#L322

It's C++ and the test is not UI, but the idea is still there.